File: src/main/java/com/linebot/model/response/RestApiResponse.java
package com.linebot.model.response;

import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
public class RestApiResponse {
    public RestApiResponse(String status) {
        this.status = status;
    }

    private String status;
}

File: src/main/kotlin/com/linebot/model/response/RestApiResponse.kt
package com.linebot.model.response

data class RestApiResponse(
        var status: String? = null
) {
    companion object {
        @JvmStatic
        fun builder(): RestApiResponseBuilder = RestApiResponseBuilder()
    }

    class RestApiResponseBuilder {
        var status: String? = null

        fun build(): RestApiResponse {
            return RestApiResponse(status)
        }

        fun status(status: String?) = apply { this.status = status }
    }
}
